python/setup.py

```diff
@@ -29,11 +29,12 @@
     long_description=long_description,
     url="http://github.com/kotekan/kotekan/",
     packages=find_packages(),
+    py_modules=["_version"],
```

Comment:
You should move `_version.py` into the`python/kotekan` directory, instead of adding this line here. That is the standard approach and will make it available to things wanting to check the version.

Too facilitate that you'll also need to edit `setup.cfg` and change two lines to:
```
versionfile_source = kotekan/_version.py
versionfile_build = kotekan/_version.py
```

Finally, I don't quite understand why we aren't sourcing the kotekan python package version from all this versioneer stuff we have merged. The standard approach to that would be to change `python/kotekan/__init__.py` to:
```
from ._version import get_versions

__version__ = get_versions()["version"]
del get_versions
```

Reply:
I've made those changes. Can you have a look, @jrs65? Should the RFI receiver script be using:
```
from ._version import get_versions

__version__ = get_versions()["version"]
del get_versions
```
instead? Which would mean that `carillon` needs to have python kotekan installed on it?

Reply:
For simplicity I would always bundle the RFI receiver script and kotekan together. Then the RFI receiver script can just do: `from kotekan import __version__` to get its version.

Reply:
I've updated to the RFI receiver script to pull the version from kotekan. All I need to do now is update `ch_ansible` so that `kotekan_python` is installed before the RFI receiver script.

Reply:
@james-s-willis you should be able to just add the `kotekan_python` role (which will probably install the script you want as it installs anyway). However, I need to quickly fix the install issue of that role because of the pip, versioneer combo. I'll let you know when that's done.

Reply:
@jrs65 yeah i just tried that and it works :+1: